packages/editor-ui/src/App.vue

```diff
@@ -160,8 +160,7 @@ export default mixins(
 		this.$externalHooks().run('app.mount');
 
 		if (this.defaultLocale !== 'en') {
-			const headers = await this.restApi().getNodeTranslationHeaders();
-			if (headers) addHeaders(headers, this.defaultLocale);
+			this.$store.dispatch('nodeTypes/getNodeTranslationHeaders');
```

Comment:
I guess we don't need to await here anymore? could set `void` before to make it clear.

Reply:
We would need `await` if something else needs to happen after, but this is the last line. Added `void`.